ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatText.qml

```diff
@@ -44,12 +44,20 @@ Item {
         clip: true
         onLinkActivated: function (link) {
             if(link.startsWith("#")) {
+                const channelName = link.substring(1);
```

Comment:
Can you put a comment here to explain what this does. I guess it's to remove the `#`?

Reply:
It is, but not sure if a comment is needed because the purpose of the substring can be infered from the prev line (the `startsWith` as well as the name of the variable `channelName`